modules/microshift-kubeconfig-local-access.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_networking/microshift-kubeconfig.adoc
+
+:_content-type: CONCEPT
+[id="local-access_{context}"]
+= Local access kubeconfig file
+
+The local access `kubeconfig` file is generates in the `/var/lib/microshift/resources/kubeadmin/kubeconfig` directory. This configuration file allows access to the local API server using `localhost` and the internal CA of `localhost`. The CA of the local access `kubeconfig` file cannot be used outside of the {product-title} host.
+
+When the cluster within the {product-title} host connects to the API server locally, a certificate is used for validation. The certificate lists `localhost` as the only Subject Alternative Name (SAN). You must use `localhost` to connect to the local API server. The `kubeconfig` file must use the same certificate authority (CA) that signed the certificate.
+
+Local access `kubeconfig` file values are taken from {product-title} built-in default values or a `config.yaml` file. You cannot add additional values, such as SANs, to the local access `kubeconfig` file. To add SANs to your deployment, you must use the remote access `kubeconfig` file or add them to a `config.yaml` file.
```

Comment:
I don't understand what the first sentence here means. What values are taking from `config.yaml`?

Maybe instead of focusing on what users can and cannot do with specific files, this would be clearer if we showed them how to do things we think they are going to want to do?

* Access the API server from the host running MicroShift
* Access the API server from another host using the IP of the MicroShift host
* Access the API server from another host using the hostname of the MicroShift host

What else?

Reply:
@dhellmann Would this be a good spot to call the other access modules we were discussing in https://issues.redhat.com/browse/OCPBUGS-10029?

Reply:
@pacevedom What do you think?